js/site_manager.js

```diff
@@ -294,6 +295,42 @@ class SiteManager {
     });
   }
 
+  async iOSbackgroundRefresh() {
+    const results = await Promise.all(
+      this.sites.map(site => site.refresh({bgTask: true})),
```

Comment:
We could probably consolidate and run the same refresh action in the non-background-task site refresh. I'd like to try that next, but for now I am limiting scope here to reduce possibility of other issues.